Mathlib/LinearAlgebra/TensorAlgebra/Basic.lean

```diff
@@ -174,20 +174,25 @@ theorem induction {C : TensorAlgebra R M → Prop}
   -- the arguments are enough to construct a subalgebra, and a mapping into it from M
   let s : Subalgebra R (TensorAlgebra R M) :=
     { carrier := C
-      mul_mem' := h_mul
-      add_mem' := h_add
+      mul_mem' := @h_mul
+      add_mem' := @h_add
       algebraMap_mem' := h_grade0 }
-  let of : M →ₗ[R] s := (ι R).codRestrict s.to_submodule h_grade1
+  -- porting note: split `let` in two to isolate a timeout in the second half
+  let h : AddCommMonoid s := inferInstanceAs (AddCommMonoid (Subalgebra.toSubmodule s))
+  let of' : M →ₗ[R] (Subalgebra.toSubmodule s) :=
+    (ι R).codRestrict (Subalgebra.toSubmodule s) h_grade1
```

Comment:
There are 3 let statements now. Is even the `let h` needed?
Is there a zulip discussion about this problem?

Reply:
`h` is needed for `of`. I added a porting note for this.